install.md

```diff
@@ -1,5 +0,0 @@
-# libpod Installation Instructions
```

Comment:
I still see value in keeping this file and let it point to podman.io as it directs users arriving on GitHub to the right location.

Reply:
Yes leave this file and put the link into it.

Reply:
Yes please do leave the file, and that will likely cure your RPM test issue below. 
Otherwise LGTM